### ISSUE TITLE
CRLF env files breaks iOS build

### ISSUE THREAD

**Issue by Aidurber** [luggit/react-native-config#201]
I've been pulling my hair out trying to get env files working with iOS. 

It throws the following errors:
```
In file included from /Users/dev/code/app/node_modules/react-native-config/ios/ReactNativeConfig/ReactNativeConfig.m:2:
/Users/dev/code/app/ios/build/Build/Products/GeneratedDotEnv.m:1:34: warning: missing terminating '"' character [-Winvalid-pp-token]
  #define DOT_ENV @{ @"API_URL":@"https://api.testapi.com/api/

                                 ^
```
and
```
/Users/dev/code/app/ios/build/Build/Products/GeneratedDotEnv.m:2:1: error: expected identifier or '('
",@"APP_TITLE":@"App (Dev)" };
^
```

The BuildDotenvConfig.ruby builds the following GeneratedDotEnv.m file: 
```
  #define DOT_ENV @{ @"API_URL":@"https://api.testapi.com/api/
",@"APP_TITLE":@"App (Dev)" };
```

Changing from CRLF to LF fixes it. I'm not knowledgable of ruby so I won't know where to start with a PR for the fix. I guess it just needs to trim the carriage return.

**Comment by OisinOKeeffe** [comment]
line 59 in
https://github.com/luggit/react-native-config/blob/master/ios/ReactNativeConfig/BuildDotenvConfig.ruby

change:
`dotenv_objc = dotenv.map { |k, v| %Q(@"#{k}":@"#{v}") }.join(",")`

to:
dotenv_objc = dotenv.map { |k, v| %Q(@"#{k}":@"#{v.chomp}") }.join(",")

**Comment by pedro** [comment]
Fixed in the pull above, released with v0.11.5! Thanks folks
